tests/OAuth2SummitSelectedPresentationListApiTest.php

```
@@ -375,13 +399,154 @@ public function testAddIndividualSelectionListAndReorderRemove(){
+        $this->assertResponseStatus(200);
+        $list = json_decode($content);
+        $this->assertTrue(!is_null($list));
+        return $list;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function testAddAndRemovePresentationFromIndividualList(){
+
+        $selection_list = $this->testAddIndividualSelectionListAndAddSelection();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$selection_list'. (undefined)

(UnusedLocalVariable)